packages/twig-to-php-parser/lib/twig-to-php-parser.php

```diff
@@ -141,6 +141,10 @@ function twig_to_php_parser( $patterns_dir_path, $template_dir_path, $is_using_p
 			if ( $has_filter ) {
 				$string_parts  = explode( '|', $variable_name );
 				$variable_name = $string_parts[0];
+
+				if ( 'markup' === $string_parts[1] ) {
```

Comment:
@ethitter Can you add a test for this in packages/twig-to-php-parser/__tests__?

This would probably entail adding a Twig file to fixtures/src/modules with this syntax and the expected PHP output to fixtures/template-parts-expected/patterns/modules.

Reply:
@laras126 Test has been added!